Open_Vuln_Score_History/OVSH.py

```diff
@@ -0,0 +1,110 @@
+import requests
+import pandas as pd 
+import json
+import gzip
+import time
+import re
+from tqdm.auto import tqdm
+
+# Small Demo
+RiskToken = "PasteAPIKeyHere"
+
+# Setup Data Dump
```

Comment:
This is uber nit-picky, but you could pull these sections out into their own functions so you don't need the comments. That'll make the code self documenting and if someone updates it in the future, they won't have to make sure all the comments are still valid. Since this is a script though, I don't think its crazy to just have everything on the top level.

Reply:
WIll work on this on V2.